### ISSUE TITLE
inspec check warnings for V-76703 

### ISSUE THREAD

**Issue by djhaynes** [mitre/microsoft-iis-8.5-server-stig-baseline#32]
inspec check outputs warnings for V-76703.

* microsoft-iis-8.5-server-stig-baseline/controls/V-76703.rb:1: Control V-76703 has no title
* microsoft-iis-8.5-server-stig-baseline/controls/V-76703.rb:1: Control V-76703 has no descriptions

However, that control appears to have a title and description so I am not sure what is causing this.

**Comment by HackerShark** [comment]
This issue is caused because the title is calling an old attribute which no longer exists. Another issue was created finding the attribute error and updating it to call the correct new attribute. Issue #25 logs that issue and PR #30 is the fixed version of the code. Once that is merged into master you should no longer see any inspec check errors.

I will close this issue since it is covered by another current open issue.
